src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,127 @@
+package duke.tasklist;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+
+
+/**
+ * TaskList class that contains arraylist of tasks
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Empty constructor of TaskList that initializes tasks
+     */
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+
+    /**
+     * Constructor that initializes tasks
+     *
+     * @param tasks takes in tasks and stores it here
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Getter that gets tasks
+     *
+     * @return tasks
+     */
+    public ArrayList<Task> getArray() {
+        return tasks;
+    }
+
+    /**
+     * Adds tasks
+     *
+     * @param task the task to be added to tasks
+     */
+    public void add(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Removes tasks
+     *
+     * @param integer the index of the item that is to be removed
+     */
+    public void remove(int integer) {
+        tasks.remove(integer - 1);
+    }
+
+    /**
+     * Returns the last task
+     *
+     * @return the last task in the tasks arraylist
+     */
+    public Task latestTask() {
+        return tasks.get(tasks.size() - 1);
+    }
+
+    /**
+     * Returns the size of the taskList
+     *
+     * @return the size of the taskList
+     */
+    public int taskListSize() {
```

Comment:
Perhaps replacing this method name with a getSize may make this clearer to other developers